ArmPkg/Drivers/MmCommunicationPei/MmCommunicationPei.inf

```diff
@@ -0,0 +1,40 @@
+#/** @file -- MmCommunicationPei.inf
+#
+#  Copyright (c) 2011-2014, ARM Ltd. All rights reserved.<BR>

```

Comment:
Is this really straight from some old Arm code?

Reply:
The other files in the module have "Copyright (c) Microsoft Corporation. All rights reserved." so wanted to double check.

Reply:
Thanks. The copyrights are fixed.

Reply:
The MSFT copyrights and the original ones are both updated/added.